definitions/variable/technology/electricity-grid.yaml

```diff
@@ -28,13 +28,27 @@
 - Network|Electricity|Active Power Flow:
     description: Active power flow through a line
     unit: MW
+- Network|Electricity|Active Power Flow Generation Contribution:
+    description: The active power flow that a generator is causing on a line
+    unit: MW
+- Network|Electricity|Active Power Flow Demand Contribution:
+    description: The active power flow that a demand is causing on a line
+    unit: MW
 - Network|Electricity|Impedance:
     description: Impedance of an interconnection between 2 regions
     unit: S
 - Network|Electricity|Investment:
     description: Overnight investment (capital) cost in transmission lines of the
       power network
     unit: [million EUR_2020/yr, billion USD_2010/yr]
+- Network|Electricity|Generation Allocated Investment:
+    description: Overnight investment (capital) cost in a transmission line of the
+      power network allocated to a generator based on its electrical usage
+    unit: [thousand EUR_2020/yr, thousand USD_2010/yr]
```

Comment:
Who is really going to pay for this? I wouldn't think it is the generator, no? More the entity who is using the interco to send electricity from region 3 to somewhere via region1>region2 interco?